lib/checkPullRequestLabels.js

```diff
@@ -1,3 +1,47 @@
+var matchChangelogLabelWithRegex = function(changelogLabel) {
+  // Please refer to https://regex101.com/r/Eo976S/2 for explanation
+  // of the below regular expression.
+  var regExpForChangelogLabel = (
+    /^([cC][hH][aA][nN][gG][eE][lL][oO][gG]: .* -- @\s*\w+\s*)/);
+  return regExpForChangelogLabel.test(changelogLabel);
+}
+
+module.exports.checkAssignee = async function(context) {
+  var pullRequest = context.payload.pull_request;
+  var pullRequestNumber = pullRequest.number;
+
+  // eslint-disable-next-line no-console
+  console.log('RUNNING ASSIGNEE CHECK ON PULL REQUEST ' +
+    pullRequestNumber + ' ...');
+
+  var changelogLabel = context.payload.label.name.trim();
+  var hasChangelogLabel = changelogLabel.toUpperCase().startsWith('CHANGELOG');
```

Comment:
Just to verify, this won't trigger if Oppiabot puts a PR merge conflict label right?

Reply:
No, it will only trigger if the label name starts with CHANGELOG, which is not the case for merge conflicts label.